docker/entrypoints/web.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/sh
+
+../../docker/common.sh
+
+python3 manage.py migrate
+cat ../../docker/scripts/createsuperuser.py | python3 manage.py shell
+
+# collect static in background
+python3 manage.py collectstatic --no-input &
+
+python3 manage.py loaddata test_data
```

Comment:
Doesn't this file run every time we start the docker container? I think we only want it to run once, during initial setup?

Reply:
Right. The same happens with the `migrate` and `collectstatic`. I don't have a good pattern yet. Do you have a suggestion?

Reply:
not really :/

Reply:
My idea was to use a Python script,

- to make a query and if `test` user already exists don't call `loaddata`
- test if the manifest file is on the storage and don't call `collectstatic` in that case

Finally, these commands could be forced by using an environment variable and running docker compose like,

```
RUN_MIGRATE=t RUN_COLLECTSTATIC=t RUN_LOADDATA=t docker-compose up
```